packages/ui-extensions-react/tsconfig.json

```diff
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "outDir": "build/ts",
     "rootDir": "src",
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "stripInternal": true
```

Comment:
Missed this when moving back to unstable. We had in internal already.

But not for the non-react package. I've also added it here. I don't think it makes sense to only strip internals for react.